src/components/DateFilter/index.tsx

```diff
@@ -265,6 +240,63 @@ const DateFilter: React.FC<PropsWithChildren> = () => {
     'December',
   ];
 
+  const [startTime, setStartTime] = useState('00:00');
+  const [endTime, setEndTime] = useState('23:59');
+
+  const formatDateWithTime = (date: Date, time: string) => {
+    const [hours, minutes] = time.split(':').map(Number);
```

Comment:
I cant edit hour/minute values, probably should be a state
![image](https://github.com/user-attachments/assets/151f2a0b-35a8-405c-8621-50382f586385)